viewer/packages/api/src/api-helpers/Image360ApiHelper.ts

```diff
@@ -121,6 +130,7 @@ export class Image360ApiHelper {
     this._domElement.addEventListener('keydown', this._domEventHandlers.exit360ImageOnEscapeKey);
 
     this._requestRedraw();
+    this._imageCollection!.events.image360Entered.fire();
```

Comment:
This will not work sadly :( 

This use case would brake:
1. User adds a collection A
2. User adds a collection B
3. User clicks an image from Collection A -> event gets fired for collection B :(